lib/router/handler-info.js

```diff
@@ -24,7 +29,60 @@ function HandlerInfo(_props) {
 
 HandlerInfo.prototype = {
   name: null,
-  handler: null,
+
+  getHandler: function() {},
+
+  _handler: DEFAULT_HANDLER,
```

Comment:
I think this needs to be in the constructor itself.  We want to shape the instance not the prototype...